js/components/adPanel.js

```diff
@@ -8,11 +8,11 @@
  * @constructor
  */
 var React = require('react'),
-  CONSTANTS = require('../constants/constants'),
-  Spinner = require('./spinner'),
-  ClassNames = require('classnames'),
-  Utils = require('./utils'),
-  Icon = require('../components/icon');
+    CONSTANTS = require('../constants/constants'),
```

Comment:
I don't know if my other comment got erased.  

@aeng7  What do you think if we enforce having to write 'var' for all new variable declarations instead of the this comma separated style?

It would use this rule I think: 
one-var: ["error", "never"]
From: https://eslint.org/docs/rules/one-var

Reply:
I personally like to split the declarations and it could be better to split them if we plan on moving to const.

Reply:
Although it seems it can't be autofixed https://github.com/eslint/eslint/issues/9072

Reply:
Let's not put that in and we'll just update the standards for now.  Maybe one of us can write a script to fix it up.

Reply:
Actually, can you put that rule in as a warning?